test/helpers/add-pr-late-label.test.ts

```diff
@@ -0,0 +1,112 @@
+/*
```

Comment:
Check out [this example](https://github.com/ExpediaGroup/github-helpers/blob/main/test/helpers/remove-pr-from-merge-queue.test.ts#L36) for how to mock the `Date` object